docs/pages/getting-started/4-connecting-crm.rst

```diff
@@ -0,0 +1,216 @@
+.. _connecting-crm:
+
+Getting Started on customer integration
+#######################################
+
+This getting started manual assumes a vendor sells services to its customers.
+The vendor manages the presence of those customers on the Nuts network through the Nuts registry.
+It's very likely the vendor has software to manage customer environments. We'll call it a *CRM* where the *customers* part relates to organizations and not people.
+This does not mean that a stand-alone installation isn't supported. In that case the vendor and organization are the same.
+
+The Nuts registry enables service discovery for organizations. The registry identifies organizations through their :ref:`DID <did>`.
+The DIDs are new and do not match the internal identifiers known to the CRM.
+This connection has to be made in order for the CRM to make adjustments to DID Documents and issue Verifiable Credentials.
+
+All APIs used in the following chapters are documented at the :ref:`API <nuts-node-api>` page.
+Open API Spec files are available for generating client code.
+
+Vendor integration
+******************
+
+As a vendor, you're in power of:
+
+- running a Nuts node
+- handling organization key material
+- updating the organization DID Document
+- defining service endpoints
+- issuing name credentials for organizations
+- trusting other vendors
+
+The last three points require a setup where a vendor DID is created. This DID will act as the controller of all organization DID Documents.
+This will allow for reuse of service endpoints and issuing of verifiable credentials. It's easier for other vendors to trust a single issuer DID.
```

Comment:
> It's easier for other vendors to trust a single issuer DID.

Sounds like it's missing something. Something like "Since every DID issuing Verifiable Credentials must be trusted individually, it's easier for other vendors when the vendor uses a single DID for issuing credentials."